### DISCUSSION TITLE
astro vercel not running npm run build, giving stale data on deploy

### DISCUSSION THREAD

**Discussion by Bijig0** [vercel/community#7310] [Help]
### Summary

Hey guys, I have an astro site deployed to vercel. It's connected to a cms which means it fetches remote data. Now I get that I need to re-build each time to see the new data, but vercel deploy apparently doesn't run npm run build? If I build on my local machine then deploy to Vercel then it works with the new data since npm run build on my machine recompiles everything and refetches data from the CMS, but I'm expecting the same behaviour when deploying to Vercle which I'm not seeing.

This is my build log in Vercel
<img width="1203" alt="Screenshot_2024-06-28_at_10 45 41_AM" src="https://github.com/vercel/community/assets/76978374/7e543f2e-d301-4573-8b35-fd2295b6a77a">

### Steps to Reproduce

Run an astro starter project on vercel

**Comment by leerob** [comment]
Hey there, can you create a minimal reproduction of the issue so we can take a look? 

https://vercel.com/guides/creating-a-minimal-reproducible-example

[a comment by an automated account is not shown]
